### ISSUE TITLE
style Twitter widget

### ISSUE THREAD

**Issue by J0WI** [mozillacommunitych/mozilla.ch#33]
How should the Twitter feed looks like?

**Comment by freaktechnik** [comment]
I'd suggest one two three tweets, similar to how the twitter widget on mozilla.org looks.

**Comment by J0WI** [comment]
The iframe has very limited options (https://dev.twitter.com/web/embedded-timelines#options). I already tried to set a limit, but this results in scrolling issues. So I think we need the API anyway.

**Comment by colinfrei** [comment]
We have the API call and should have the data, just need to decide what exactly we want to display and how.

@freaktechnik you mean the tile that displays one tweet on mozilla.org?
I think that'd work fine. We could make it a bit bigger, since twitter is one of our main communication channels, but that may just be to display one tweet and make the text bigger/include an image if present.

**Comment by freaktechnik** [comment]
@colinfrei exactly. I think we have the space to display multiple tweets. I'm not sure how easy the image handling is with the data we get. But we can always replace a tweet with an image in that box.

**Comment by MichaelKohler** [comment]
could we have something like this:
1) if the last tweet contains an image, only show this one
2) if it doesn't show the last two tweets without picture

@colinfrei does the current implementation on the API side provide this information?

**Comment by colinfrei** [comment]
We cache the last 5 tweets - I assume that includes the all that information. We don't currently download any attached images, that would be an improvement :)

**Comment by MichaelKohler** [comment]
who can work on this?

**Comment by MichaelKohler** [comment]
We would still take a custom Twitter tile instead of the normal Twitter one ;)

**Comment by MichaelKohler** [comment]
@lkappeler still interested in this?
